src/fileActions.jsx

```diff
@@ -124,7 +124,7 @@ export const ConfirmDeletionDialog = ({
             modalTitle = cockpit.format(_("Delete file $0?"), selectedItem.name);
         } else if (selectedItem.type === "lnk") {
             modalTitle = cockpit.format(_("Delete link $0?"), selectedItem.name);
-        } else if (selectedItem.type === "dir" || selectedItem.items_cnt) {
+        } else if (selectedItem.type === "dir") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-256-20240212-120023-29378c53-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)